### ISSUE TITLE
Compilation error for version 7.6.0

### ISSUE THREAD

**Issue by condor-cp** [jtv/libpqxx#501]
Hello, I wanted to install libpqxx and I took the latest tag 7.6.0 according to your documentation.
But I got the following error while building :
```
/libsrc/libpqxx/src/strconv.cxx: In function 'constexpr T {anonymous}::safe_multiply_by_ten(T)':
/libsrc/libpqxx/src/strconv.cxx:327:22: error: 'inline' specifier invalid for variable 'ten' declared at block scope
   inline constexpr T ten{10};
                      ^~~
/libsrc/libpqxx/src/strconv.cxx:328:61: error: 'inline' specifier invalid for variable 'high_threshold' declared at block scope
   inline constexpr T high_threshold(std::numeric_limits<T>::max() / ten);
                                                             ^~~
/libsrc/libpqxx/src/strconv.cxx:334:62: error: 'inline' specifier invalid for variable 'low_threshold' declared at block scope
     inline constexpr T low_threshold(std::numeric_limits<T>::min() / ten);
```

From https://stackoverflow.com/questions/16530970/inline-in-block-scope-function-declaration this error seems to be accepted by some compilers but not all.
I checked, the "inline" appeared between 7.5.2 and 7.6.0 and seems to be gone in the master branch. I will use 7.5.2 for now. Maybe can you do a fix to 7.6.0, removing the three bad "inline" tags  ?

Thank you

**Comment by KayEss** [comment]
The inlines are perfectly valid on values like this. What compiler (and version) are you using and what language version are you compiling with?

That said, I don't think any `inline` is actually needed as all of the symbols in that file are limited to that one file anyway

**Comment by condor-cp** [comment]
My compiler version : 
gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0

**Comment by tt4g** [comment]
Already fixed this problem by #468 in current branch.

**Comment by jtv** [comment]
Thanks folks.  Yes, gcc 7 is pretty old hat now, and my current desktop no longer even supports it.  That makes it harder to detect these casual incompatibilities as I develop.

I'm also working my way towards an 8.0 release that will require C++20 support, though we're not quite at that point yet.  A 7.7 is shaping up and after that...  well you never know what happens.  But there's no getting around it: that gcc version baseline is creeping up.  It may even leap.
